source/pages/practices/best_practices.rst

```diff
@@ -33,51 +22,47 @@ If you think your file format should be accepted by the repository, but is curre
+------------------
 
-.. _metadata:
+Data documentation ensures that research data is understood and therefore used by current and future users (including the researcher). It is vital to store data in a structured and consistent way with appropriate data documentation. The documentation can be a description explaining what the data is, what you can do with it and how it can be used. In some cases addtional technical documentation about the exact file format could be of use. Documentation can be provided in any of the accepted or preferred file formats.
 
 Metadata
-_________________
+--------
 
-Metadata provides structured information about the data. Depending on the type of data, there are different types of metadata:
+Metadata provides structured information about the data. There are different types of metadata:
 
-- **Descriptive metadata** describes and identifies information resources (minimal metdata required to find a digital object). In includes elements such as title, abstract, author, and keywords
+- **Descriptive metadata** describes and identifies information resource. It is the minimal metdata required to find a digital object and may include elements such as title, abstract, author, and keywords
 
 - **Structural metadata** provides information about the internal structure of resources including page, section, chapter numbering, indexes, and relations to other digital objects.
 
-- **Technical metadata** provides information on the technical aspects of the datasets such as data formats, hardware/software used, calibration, version, authentication, encryption.
+- **Technical metadata** provides information on the technical aspects of the dataset such as data formats, hardware/software used, calibration, version, authentication, and encryption.
 
-- **Administrative metadata** provides information on user rights and management of digital objects such as license, rights management, and access control.
+- **Administrative metadata** provides information on user rights and management of digital objects and inlcudes elements such as license, rights management, and access control.
 
-The repository will only ask for descriptive metadata during the deposit workflow, but it is highly recommended to add links to additional documentation or files containing metadata of any kind. Any other kinds of metadata are automatically added, though not always visible for the end-user.
+The repository only requires descriptive metadata during the deposit workflow. However, it is highly recommended to add links to additional documentation or files containing metadata of any kind. Any other kinds of metadata are automatically added, though not always visible for the end-user.
```

Comment:
> Any other kinds of metadata are automatically added, though not always visible for the end-user.

This sentence is not entirely clear to me, what are we trying to say?

Reply:
Some descriptive fields are automatically added by SDR and do not need to be added by the user.